packages/amplication-prisma-db/prisma/migrations/20220524135837_from_app_to_resource/migration.sql

```diff
@@ -0,0 +1,59 @@
+-- App work
+ALTER TABLE "App"
+RENAME TO "Resource";
+ALTER TABLE "Resource" RENAME CONSTRAINT  "App_pkey" TO "Resource_pkey";
+ALTER TABLE "Resource" RENAME CONSTRAINT  "App_workspaceId_fkey" TO "Resource_workspaceId_fkey";
+ALTER TABLE "Resource" RENAME CONSTRAINT  "App_projectId_fkey" TO "Resource_projectId_fkey";
+ALTER INDEX "App.workspaceId_name_unique" RENAME TO "Resource.workspaceId_name_unique";
+
+-- GitRepository work
+ALTER TABLE "GitRepository"
+RENAME "appId" TO "resource";
```

Comment:
is that correct? it was appId before and now just resource (without id)

Reply:
You are right! thanks